### ISSUE TITLE
Recent fixes aren't reflected in latest release

### ISSUE THREAD

**Issue by jkjustjoshing** [iKonrad/gatsby-source-directus#7]
@iKonrad PR #4 was merged to fix issue #3, but there is no release that includes the fix to this. 

Can you create a release so I can reference it in my package.json? (my project has a restriction that doesn't allow us to directly target commit hashes, otherwise I'd just target c7c97e0d6).

**Comment by samis-dv** [comment]
@jkjustjoshing  Was this issue fixed by any chance? Because I am still seeing the error as described in the previous issue request.

![image](https://user-images.githubusercontent.com/59620753/71943661-36972b00-3176-11ea-98e0-fe4dacc08695.png)

**Comment by jkjustjoshing** [comment]
@samis-dv I don't believe a new version has been released - you'll need to create your own release that includes PR #4 locally and then reference that.

Unfortunately I'm not using Gatsby right now so I won't be able to give you much more guidance than that.
